hles/transformation/src/test/scala/org/broadinstitute/monster/dap/environment/TemperaturePrecipitationTransformationsSpec.scala

```diff
@@ -0,0 +1,223 @@
+      "tp_tmax_annual_05" -> Array("64.6"),
+      "tp_tmax_annual_06" -> Array("66.5"),
+      "tp_tmax_annual_07" -> Array("69.7"),
+      "tp_tmax_annual_08" -> Array("73"),
+      "tp_tmax_annual_09" -> Array("63.6"),
+      "tp_tmax_annual_10" -> Array("52.5"),
+      "tp_tmax_annual_11" -> Array("47.9"),
+      "tp_tmax_annual_12" -> Array("42"),
+      "tp_tmin_annual_01" -> Array("33.9"),
+      "tp_tmin_annual_02" -> Array("25"),
+      "tp_tmin_annual_03" -> Array("32.2"),
+      "tp_tmin_annual_04" -> Array("38.5"),
+      "tp_tmin_annual_05" -> Array("45.2"),
+      "tp_tmin_annual_06" -> Array("47.5"),
+      "tp_tmin_annual_07" -> Array("52.1"),
+      "tp_tmin_annual_08" -> Array("54.1"),
+      "tp_tmin_annual_09" -> Array("49.6"),
+      "tp_tmin_annual_10" -> Array("37.6"),
+      "tp_tmin_annual_11" -> Array("35.4"),
+      "tp_tmin_annual_12" -> Array("34.3"),
+      "tp_norm_data_year" -> Array("1"),
+      "tp_pcpn_norm_01" -> Array("10.55"),
+      "tp_pcpn_norm_02" -> Array("6.92"),
+      "tp_pcpn_norm_03" -> Array("7.42"),
+      "tp_pcpn_norm_04" -> Array("5.65"),
+      "tp_pcpn_norm_05" -> Array("4.46"),
+      "tp_pcpn_norm_06" -> Array("3.47"),
+      "tp_pcpn_norm_07" -> Array("1.6"),
+      "tp_pcpn_norm_08" -> Array("1.65"),
+      "tp_pcpn_norm_09" -> Array("3.19"),
+      "tp_pcpn_norm_10" -> Array("6.83"),
+      "tp_pcpn_norm_11" -> Array("12.05"),
+      "tp_pcpn_norm_12" -> Array("9.73"),
+      "tp_tmpc_norm_01" -> Array("36.4"),
+      "tp_tmpc_norm_02" -> Array("37.9"),
+      "tp_tmpc_norm_03" -> Array("41.2"),
+      "tp_tmpc_norm_04" -> Array("45.2"),
+      "tp_tmpc_norm_05" -> Array("50.9"),
+      "tp_tmpc_norm_06" -> Array("55.9"),
+      "tp_tmpc_norm_07" -> Array("61.5"),
+      "tp_tmpc_norm_08" -> Array("62"),
+      "tp_tmpc_norm_09" -> Array("56.9"),
+      "tp_tmpc_norm_10" -> Array("48"),
+      "tp_tmpc_norm_11" -> Array("40"),
+      "tp_tmpc_norm_12" -> Array("34.9"),
+      "tp_tmax_norm_01" -> Array("41.2"),
+      "tp_tmax_norm_02" -> Array("44.3"),
+      "tp_tmax_norm_03" -> Array("48.6"),
+      "tp_tmax_norm_04" -> Array("53.8"),
+      "tp_tmax_norm_05" -> Array("60"),
+      "tp_tmax_norm_06" -> Array("65.1"),
+      "tp_tmax_norm_07" -> Array("71.9"),
+      "tp_tmax_norm_08" -> Array("72.5"),
+      "tp_tmax_norm_09" -> Array("66.7"),
+      "tp_tmax_norm_10" -> Array("55.5"),
+      "tp_tmax_norm_11" -> Array("45.1"),
+      "tp_tmax_norm_12" -> Array("39.4"),
+      "tp_tmin_norm_01" -> Array("31.6"),
+      "tp_tmin_norm_02" -> Array("31.5"),
+      "tp_tmin_norm_03" -> Array("33.7"),
+      "tp_tmin_norm_04" -> Array("36.6"),
+      "tp_tmin_norm_05" -> Array("41.7"),
+      "tp_tmin_norm_06" -> Array("46.6"),
+      "tp_tmin_norm_07" -> Array("51"),
+      "tp_tmin_norm_08" -> Array("51.4"),
+      "tp_tmin_norm_09" -> Array("47.1"),
+      "tp_tmin_norm_10" -> Array("40.5"),
+      "tp_tmin_norm_11" -> Array("34.9"),
+      "tp_tmin_norm_12" -> Array("30.3")
```

Comment:
at least in the specs, i think we should definitely be building these massive maps programmatically